carbonfive/spraygun-react-ts
# Remove mini-css-extract-plugin resolution and upgrade

Problem
=======

We had a resolution specified in our package.json for `mini-css-extract-plugin` as a temporary workaround for an incompatibility with `react-scripts`. A new version of `mini-css-extract-plugin` has since been released that [fixes the issue](https://github.com/webpack-contrib/mini-css-extract-plugin/issues/896#issuecomment-1014458314). This workaround is no longer needed.

Solution
========

1. Remove the resolution for `mini-css-extract-plugin`
1. Run `yarn upgrade`

Steps to Verify
---------------

1. Run `yarn install`
1. Run `yarn build` to confirm the app can be built without errors
